docsrc/markdown/datafeed.md

```diff
@@ -183,7 +183,7 @@ async def run():
         await datahose_loop.start()
 ```
 
-An example of the usage of the Datahose service can be found [here](../../examples/datafeed/datahose.py )
+An example of the usage of the Datahose service can be found [here](https://github.com/finos/symphony-bdk-python/tree/main/examples/datafeed/datahose.py)
```

Comment:
can't we keep relative links?

Reply:
I don't think we can do it since this document will be open on github page which is different from the repo:  https://symphony-bdk-python.finos.org/markdown/datafeed.html